test/services/pin-state-transition/hooks/prepare-activity-log.test.js

```diff
@@ -11,14 +11,21 @@ const Pin = require('../../../../src/services/pin/pin-model');
 // Fixtures
 const departments = require('../../../fixtures/departments');
 const pins = require('../../../fixtures/pins');
-const DEPARTMENT_GENERAL_ID = require('../../../fixtures/constants').DEPARTMENT_GENERAL_ID;
-const ORGANIZATION_ID = require('../../../fixtures/constants').ORGANIZATION_ID;
-const PIN_PENDING_ID = require('../../../fixtures/constants').PIN_PENDING_ID;
+
+// Constants
+const {
+  DEPARTMENT_GENERAL_ID,
+  ORGANIZATION_ID,
+  PIN_PENDING_ID,
+} = require('../../../fixtures/constants');
+const DEPARTMENT_GENERAL_OBJECT = require('../../../fixtures/departments')[1];
```

Comment:
This will cause flaky test if the order is changed in the future. Use `filter` with ObjectId or name instead.